src/main/java/seedu/address/logic/parser/reminder/FindCommandParser.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.logic.parser.reminder;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REMINDER_STATUS;
+
+import seedu.address.logic.commands.reminder.FindCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.logic.parser.ArgumentTokenizer;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FindCommand object for Reminder
+ */
+public class FindCommandParser implements Parser<FindCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindCommand
+     * and returns an FindCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    @Override
+    public FindCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_REMINDER_STATUS);
+
+        try {
+            Boolean completed;
+            if (argMultimap.getValue(PREFIX_REMINDER_STATUS).isPresent()) {
+                completed = ParserUtil.parseCompletionStatus(argMultimap.getValue(PREFIX_REMINDER_STATUS).get());
+            } else {
+                throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+            }
+            return new FindCommand(completed);
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
```

Comment:
Is this part necessary? The part where you throw an exception, then catch it, and throw it again